### ISSUE TITLE
Search Results wer ehard to click

### ISSUE THREAD

**Issue by cam156** [psu-stewardship/etda#122]
link and heading on different lines which made it difficult to know when an item was selected

**Comment by cam156** [comment]
Changing to 1.1 milestone.  It looks like blacklight has made some upgrades for accessibility and we should upgrade versions with the next release.  Since this was only a slight hassle I think it would be better to upgrade later vs adding more overrides
